src/main/java/org/jenkinsci/plugins/oic/OicSecurityRealm.java

```diff
@@ -1394,6 +1409,25 @@ public boolean handleTokenExpiration(HttpServletRequest httpRequest, HttpServlet
 
         User user = User.get2(authentication);
 
+        if (isTraditionalApiTokenAccessEnabled()) {
+            // check if this is a valid api token based request
+            if (httpRequest.getHeader("Authorization") != null) {
+                String token = new String(
+                                Base64.getDecoder()
+                                        .decode(httpRequest
+                                                .getHeader("Authorization")
```

Comment:
Should check the expected format, you never now.
```suggestion
            String authHeader = httpRequest.getHeader("Authorization");
            if (authHeader != null && authHeader.startsWith("Basic ")) {
                String token = new String(
                                Base64.getDecoder()
                                        .decode(authHeader
```